.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: python
 env:
   - PYTHON_VERSION=3.5
+  - PYTHON_VERSION=2.7
```

Comment:
Just to clarify, this is intentional? We are supporting py2?

Reply:
Well, I always thought we were but seems like due to skbio we are not ... I'm gonna delete from this PR ...